client/query_utils.py

```diff
@@ -4,7 +4,8 @@
 
 class QueryParams:
 
-    def __init__(self):
+    def __init__(self, urls):
         self.response_type = ResponseType.JSON
```

Comment:
No worries, was just wondering if there was a valid reason for this change. If so, we can keep it